org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "http://schemas.opentranslatorstothings.org/org.opent2t.sample.hub.superpopular#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "",
+  "title": "OpenT2T Hub",
+  "definitions": {
+    "org.opent2t.sample.hub.superpopular": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "ReadOnly, Gets the name of the hub."
+        },
+        "devices": {
+          "description": "ReadOnly, Gets the current set of devices associated with the hub."
+        }
```

Comment:
Add the device type.

I agree we totally would end up doing a call per property and if we're going to the cloud that could potentially be hundreds of call and doesn't scale. AllJoyn is not something I have much experience in and we should let Jason comment on ways to optimize.  I'm not sure what will need to change to fully work in the AllJoyn world, but I know we can't pass objects around so we have this limitation on the design. I expect tweaks (hopefully) once we start working on enabling AllJoyn.

---
In reply to: [76345446](https://github.com/openT2T/translators/pull/75#discussion_r76345446) [](ancestors = 76345446,76344264,76336787,76335569,76334693,76154047)